app/lib/core/models/contingency_plan_info.dart

```diff
@@ -26,6 +26,32 @@ class ContingencyPlanInfo {
         _potentialSafetyActions = potentialSafetyActions,
         _contingencyEvents = contingencyEvents;
 
+  ContingencyPlanInfo.fromJSON(Map<String, dynamic> json)
+      : goalStatement = json['goalStatement'],
+        communicationPlan = json['communicationPlan'],
+        _crisisContacts = List.from(json['_crisisContacts']),
+        expectedPrepProcess = json['expectedPrepProcess'],
+        standardAnimalMonitoring = json['standardAnimalMonitoring'],
+        _potentialHazards = List.from(json['_potentialHazards']),
+        _potentialSafetyActions = List.from(json['_potentialSafetyActions']),
+        _contingencyEvents = json['_contingencyEvents']
+            .map<ContingencyPlanEvent>((contingencyEvent) =>
+                ContingencyPlanEvent.fromJSON(contingencyEvent))
+            .toList();
+
+  Map<String, dynamic> toJSON() => {
+        'goalStatement': goalStatement,
+        'communicationPlan': communicationPlan,
+        '_crisisContacts': _crisisContacts,
```

Comment:
- [ ] No underscore, the underscore is just to define private methods in dart files. So, not necessary here.
We can also do this later, not a big deal.